libs/designsystem/tsconfig.lib.json

```diff
@@ -2,9 +2,8 @@
   "extends": "./tsconfig.json",
   "compilerOptions": {
     "outDir": "../../dist/out-tsc",
-    "target": "es2015",
+    "target": "es2020",
```

Comment:
Just for clarity, I think this is safe since major projects using us are already targetting es2020 in their configurations.

Reply:
It should be absolutely safe - even the current angular cli creates projects with this compiler target.